platform/ios/src/MGLMapView.h

```diff
@@ -296,6 +296,17 @@ MGL_EXPORT IB_DESIGNABLE
 
 #pragma mark Displaying the User’s Location
 
+/**
+ The object that this map view uses to start and stop the delivery of location-related
+ events.
+ 
+ If no custom manager is provided or setting this property to `nil` will default
+ to the internal `CLLocationManager` based implementation.
+ 
+ Set the custom location manager before calling `showUserLocation`.
+ */
+@property (nonatomic, nullable) id<MGLLocationManager> locationManager;
```

Comment:
If the developer sets this property to `nil` or leaves it as `nil`, should the getter return `nil` or should it return the default implementation? If the latter, then the property should be `null_resettable` instead of `nullable`.

Most of the time we’ve received requests to expose the location manager, it wasn’t because the developer wanted to replace the location manager with their own implementation. Rather, it was because they wanted to tweak the built-in location manager’s settings. For that to be possible, this property would always need to return a non-nil value.